examples/SX127x/SX127x_Receive_FHSS/SX127x_Receive_FHSS.ino

```diff
@@ -0,0 +1,122 @@
+/*
+   RadioLib SX127x Transmit with Frequency Hopping Example
+
+   This example transmits packets using SX1278 LoRa radio module.
+   Each packet contains up to 256 bytes of data, in the form of:
+    - Arduino String
+    - null-terminated char array (C-string)
+    - arbitrary binary data (byte array)
+
+   Other modules from SX127x/RFM9x family can also be used.
+
+   For default module settings, see the wiki page
+   https://github.com/jgromes/RadioLib/wiki/Default-configuration#sx127xrfm9x---lora-modem
+
+   For full API reference, see the GitHub Pages
+   https://jgromes.github.io/RadioLib/
+
+   The SX1276 / 7 / 8 / 9 supports FHSS or Frequency Hopping Spread Spectrum.
+   Once a hopping period is set and a transmission is started the radio
+   will begin triggering interrupts every hop period where the radio frequency
+   is changed to the next channel. This allows a simple mechanism to abide by
+   the FCC 400ms max dwell time rule.
+   https://www.govinfo.gov/content/pkg/CFR-2019-title47-vol1/pdf/CFR-2019-title47-vol1-sec15-247.pdf
+*/
+
+#include <RadioLib.h> //Click here to get the library: http://librarymanager/All#RadioLib
+
+//Pins for RFM97 100mW Shield to SparkFun ESP32 Thing Plus C
+int pin_cs = 15;
+int pin_dio0 = 26;
+int pin_dio1 = 25;
+int pin_rst = 32;
+SX1276 radio = new Module(pin_cs, pin_dio0, pin_rst, pin_dio1);
+
+int counter = 0;
+
+volatile bool rxComplete = false;
+volatile bool fhssChange = false;
+
+//The channel frequencies can be generated randomly or hard coded
+float channels[] = {908.0, 906.0, 907.0, 905.0, 903.0, 910.0, 909.0};
+int numberOfChannels = sizeof(channels) / sizeof(float);
+
+int hopsCompleted = 0;
+
+void setup()
+{
```

Comment:
Please follow the code style guidelines: https://github.com/jgromes/RadioLib/blob/master/CONTRIBUTING.md#code-style-guidelines